x/launch/keeper/invariants.go

```diff
@@ -0,0 +1,103 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/tendermint/spn/x/launch/types"
+)
+
+// RegisterInvariants registers all module invariants
+func RegisterInvariants(ir sdk.InvariantRegistry, k Keeper) {
+	ir.RegisterRoute(types.ModuleName, "zero-launch-timestamp",
```

Comment:
Maybe we can move routes to constants and reuse in the other part of the code?

Reply:
I like the solution, but the methods are hard for we can reuse. There is too much specific logic for being reusable. I used the same pattern from the [cosmos-sdk modules](https://github.com/cosmos/cosmos-sdk/blob/master/x/bank/keeper/invariants.go).